kv/kv.go

```diff
@@ -63,12 +63,8 @@ type RetrieverMutator interface {
 	Mutator
 }
 
-// MemBuffer is an in-memory kv collection. It should be released after use.
-type MemBuffer interface {
-	RetrieverMutator
-	// Release releases the buffer.
-	Release()
-}
+// MemBuffer is an in-memory kv collection, can be used to buffer write operations.
+type MemBuffer RetrieverMutator
```

Comment:
MemBuffer indicates the underlying storage is memory, it's clearer than RetrieverMutator.